wg-model-registry/README.md

```diff
@@ -0,0 +1,28 @@
+# ML Model Registry Working Group
```

Comment:
I would probably remove this file from the proposal

Reply:
Can I understand the reason for removing the working group readme? It was created following the ODH community guidelines.

Reply:
My comment is because we are reviewing ODH community governance but I agree with you, the current proposal is aligned with what is the existing project. Fine for me to keep it